### DISCUSSION TITLE
组件的具名/默认插槽如何在文档展示？

### DISCUSSION THREAD

**Discussion by anlyyao** [Tencent/tdesign-miniprogram#3138] [Ideas]
### 背景
小程序文档目前没有单独展示 slot 插槽的地方，一般为属性自带同名插槽， 所以如果没有属性承载的不具名插槽就会被遗漏，比如 [overlay](https://tdesign.tencent.com/miniprogram/components/overlay?tab=api) 组件。

**不具名/默认 slot 如何在文档展示？？**  这里以 `Empty` 组件为例，对比其他业内小程序组件库在文档中对插槽呈现方式。

**如果你有更好的建议，欢迎留言讨论～** 

### [Vant Weapp](https://vant-ui.github.io/vant-weapp/#/empty)
<img width="1289" alt="截屏2024-09-10 18 14 52" src="https://github.com/user-attachments/assets/e0a5cd02-bbf7-4980-b02d-0efbf59a758c">

### [Ant Design Mini ](https://mini.ant.design/components/empty#api)
<img width="1477" alt="截屏2024-09-10 18 15 03" src="https://github.com/user-attachments/assets/4aa442f7-1a47-45ab-9b9b-1e23974586ee">

### [Lin UI](https://doc.mini.talelin.com/component/basic/button.html#%E6%8C%89%E9%92%AE%E6%8F%92%E6%A7%BD)
<img width="956" alt="截屏2024-10-12 15 06 40" src="https://github.com/user-attachments/assets/4294ca6e-b2bd-4fcd-9165-62f80630f0db">
<img width="969" alt="截屏2024-10-12 15 03 30" src="https://github.com/user-attachments/assets/fe5a106e-08d4-46b9-b55a-b125cfbd0aaa">

### [TDesign Miniprogram](https://tdesign.tencent.com/miniprogram/components/empty?tab=api)
<img width="1172" alt="截屏2024-09-10 18 22 56" src="https://github.com/user-attachments/assets/af3bf53e-231f-4dcf-b831-89dba4dc0452">

**Comment by jarmywang** [comment]
支持插槽单独列出来，更好的展示默认插槽和具名插槽

**Comment by uyarn** [comment]
支持单独列出来 小程序插槽方面不像其他端或者框架有约定的使用方式 确实还是需要明确标识

**Comment by anlyyao** [comment]
处理在 https://github.com/TDesignOteam/tdesign-api/pull/720
